hrjobcontract/CRM/Hrjobcontract/DAO/HRJobLeave.php

```diff
@@ -174,7 +174,7 @@ static function &fields()
               ) ,
               'hrjobcontract_leave_leave_amount' => array(
                 'name' => 'leave_amount',
-                'type' => CRM_Utils_Type::T_INT,
+                'type' => CRM_Utils_Type::T_STRING,
```

Comment:
I saw the explanation about this on the PR description, but I'm not sure of why it has to be a string and not a decimal.

Regardless of the type, if you change the DAO you have to:
- Upgrade the docblock related to this field
- Update the install SQL file
- Add an upgrader to update existing sites